config/allowlistlist.go

```diff
@@ -16,24 +17,19 @@ func NewAllowlist(udpPorts []int64, tcpPorts []int64, subnets []string) Allowlis
 		tcp[port] = true
 	}
 
-	subs := map[string]bool{}
-	for _, sub := range subnets {
-		subs[sub] = true
-	}
-
 	return Allowlist{
 		Ports: Ports{
 			UDP: udp,
 			TCP: tcp,
 		},
-		Subnets: subs,
+		Subnets: subnets,
 	}
 }
 
 // Allowlist is a collection of ports and subnets
 type Allowlist struct {
-	Ports   Ports   `json:"ports"`
-	Subnets Subnets `json:"subnets"`
+	Ports   Ports    `json:"ports"`
+	Subnets []string `json:"subnets"` // TODO change to netip.Prefix and refactor
```

Comment:
Previously this was set with amortized `O(1)` lookup, but now it's `O(n)` lookup. I'm guessing this won't have huge impact here right?

Reply:
> Previously this was set with amortized `O(1)` lookup, but now it's `O(n)` lookup. I'm guessing this won't have huge impact here right?

It should not as data set is not that big and lookup is never on the hot path. Ordering of elements is important in this case